src/Security/Certificate.cs

```diff
@@ -369,22 +440,63 @@ public NSData GetNormalizedSubjectSequence ()
 
 #if MONOMAC
 		[DllImport (Constants.SecurityLibrary)]
-		[Deprecated (PlatformName.MacOSX, 10,13)]
-		static extern /* __nullable CFDataRef */ IntPtr SecCertificateCopySerialNumber (IntPtr /* SecCertificateRef */ certificate, IntPtr /* CFErrorRef * */ error);
+#if !NET
+		[Deprecated (PlatformName.MacOSX, 10, 13)]
 #else
+		[UnsupportedOSPlatform ("macos.10.13")]
+#if MONOMAC
+		[Obsolete ("Starting with macos.10.13 ", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
```

Comment:
^ not needed (no message)

Reply:
Message should be "Use 'SecCertificateCopySerialNumberData' instead."

Reply:
Lower case Use?